src/NuGet.Licenses/Content/Site.css

```diff
@@ -24,4 +24,12 @@ pre {
 
 #license-expression > a {
     white-space: nowrap;
-}
\ No newline at end of file
+}
+
+.optional-license-text {
+    color: #0000cc;
```

Comment:
Why color value here is lowercase and couple of lines below is upper-case? :)

Reply:
I assume it passes accessibility checks.

Reply:
I copied it verbatim from the SPDX site.

a11y passes. Thanks for the reminder.